tools/serverpod_cli/lib/src/config/config.dart

```diff
@@ -240,7 +262,12 @@ class GeneratorConfig {
         serverPackageDirectoryPathParts: serverPackageDirectoryPathParts,
         relativeDartClientPackagePathParts: relativeDartClientPackagePathParts,
         modules: modules,
-        extraClasses: extraClasses);
+        extraClasses: extraClasses,
+        serializeEnumValuesAsStrings: serializeEnumValuesAsStrings);
+
```

Comment:
```dart
assert(_instance == null, 'GeneratorConfig should only be created once.');
```

Reply:
I added this to `GeneratorConfig.load` rather than the constructor, so that it's possible to still construct other `GeneratorConfig`s by calling the constructor directly, without setting `_instance`.